### ISSUE TITLE
Uncaught TypeError: Cannot read property 'items' of undefined

### ISSUE THREAD

**Issue by ApertureStudiosMedia** [abe33/atom-project-palette-finder#24]
Hi, Instantly after installing the plugin Atom Project Palette Finder, i have received this error bellow.
Hope this information helps towards fixing this bug.

Cheers.

Uncaught TypeError: Cannot read property 'items' of undefined

**Atom Version**: 0.161.0
**System**: Mac OS X 10.9.4
**Thrown From**: [project-palette-finder](https://github.com/abe33/atom-project-palette-finder) package, v2.4.9

### Steps To Reproduce

1. ...
2. ...

### Stack Trace

```
At /Users/WillieWright/.atom/packages/project-palette-finder/lib/palette-provider.coffee:18

TypeError: Cannot read property 'items' of undefined
  at PaletteProvider.buildSuggestions (/Users/WillieWright/.atom/packages/project-palette-finder/lib/palette-provider.coffee:18:25)
  at AutocompleteManager.module.exports.AutocompleteManager.runAutocompletion (/Users/WillieWright/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:180:39)
  at /Users/WillieWright/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1

```

/cc @atom/core

**Comment by imperez** [comment]
I've created Pull Request #26 to fix this issue as a graceful fallback. I don't know the codebase that well so this might not be the best solution so hopefully @abe33 can give this a proper look.

**Comment by abe33** [comment]
Fixed, I'm closing this issue.

**Comment by tststs** [comment]
Doesn't seems to be fixed. Package is @2.4.11
Occurs on every time i type something in my scss file.

TypeError: Cannot read property 'items' of undefined
  at Object.provider.requestHandler (/***/.atom/packages/project-palette-finder/lib/project-palette-finder.coffee:142:31)
  at /***/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:119:39
  at Array.map (native)
  at AutocompleteManager.module.exports.AutocompleteManager.scatterRequest (/***/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:118:27)
  at AutocompleteManager.scatterRequest (/***/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1)
  at AutocompleteManager.module.exports.AutocompleteManager.runAutocompletion (/***/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:113:6)
  at /***/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1

**Comment by abe33** [comment]
I forgot to update the code while migrating to autocomplete 2 API. Fixed in 2.4.12.

**Comment by tststs** [comment]
Looking good!

Thx
tststs
